src/main/java/microsoft/exchange/webservices/data/ExchangeServiceBase.java

```diff
@@ -593,81 +540,69 @@ protected void setCustomUserAgent(String userAgent) {
-        /*public void setCookie(URL url, String value) throws IOException,
-	URISyntaxException {
-		CookieHandler handler =CookieHandler.getDefault();
-		if (handler != null) {
-			Map<String, List<String>> headers =
-				new HashMap<String, List<String>>();
-			List<String> values = new Vector<String>();
-			values.add(value);
-			headers.put("Cookie", values);
-
-			handler.put(url.toURI(), headers);
-		}
-	}*/
-
-	/*
-	 * Gets the cookie.
-	 * 
-	 * @param url
-	 *            the url
-	 * @return the cookie
-	 * @throws java.io.IOException
-	 *             Signals that an I/O exception has occurred.
-	 * @throws java.net.URISyntaxException
-	 *             the uRI syntax exception
-	public String getCookie(URL url) throws IOException, URISyntaxException {
-		String cookieValue = null;
-
-		CookieHandler handler = CookieHandler.getDefault();
-		if (handler != null) {
-			Map<String, List<String>> headers = handler.get(url.toURI(),
-					new HashMap<String, List<String>>());
-			List<String> values = headers.get("Cookie");
-			for (Iterator<String> iter = values.iterator(); iter.hasNext();) {
-				String v = iter.next();
-
-				if (cookieValue == null)
-					cookieValue = v;
-				else
-					cookieValue = cookieValue + ";" + v;
-			}
-		}
-		return cookieValue;
-	}*/
+//  /**
+//   * Gets the cookie container. <value>The cookie container.</value>
+//   *
+//   * @param url   the url
+//   * @param value the value
+//   * @throws java.io.IOException         , URISyntaxException
+//   * @throws java.net.URISyntaxException the uRI syntax exception
+//   */
+//  public void setCookie(URL url, String value) throws IOException,
+//      URISyntaxException {
+//    CookieHandler handler = CookieHandler.getDefault();
+//    if (handler != null) {
+//      Map<String, List<String>> headers =
+//          new HashMap<String, List<String>>();
+//      List<String> values = new Vector<String>();
+//      values.add(value);
+//      headers.put("Cookie", values);
+//
+//      handler.put(url.toURI(), headers);
+//    }
+//  }
+//
+//  /**
+//   * Gets the cookie.
+//   *
+//   * @param url the url
+//   * @return the cookie
+//   * @throws java.io.IOException         Signals that an I/O exception has occurred.
+//   * @throws java.net.URISyntaxException the uRI syntax exception
+//   */
+//  public String getCookie(URL url) throws IOException, URISyntaxException {
+//    String cookieValue = null;
+//
+//    CookieHandler handler = CookieHandler.getDefault();
+//    if (handler != null) {
+//      Map<String, List<String>> headers = handler.get(url.toURI(),
+//          new HashMap<String, List<String>>());
+//      List<String> values = headers.get("Cookie");
+//      for (Iterator<String> iter = values.iterator(); iter.hasNext(); ) {
+//        String v = iter.next();
+//
+//        if (cookieValue == null)
+//          cookieValue = v;
+//        else
+//          cookieValue = cookieValue + ";" + v;
+//      }
+//    }
+//    return cookieValue;
+//  }
```

Comment:
I don't really know what's the deal with this code. There's a lot out code that has been commented out in this library, most of which could probably be removed...